src/Lean/ToExpr.lean

```diff
@@ -138,34 +139,35 @@ instance : ToExpr Name where
   toExpr     := Name.toExprAux
   toTypeExpr := mkConst ``Name
 
-instance [ToExpr α] : ToExpr (Option α) :=
+instance {α : Type u} [l : ToLevel.{u}] [ToExpr α] : ToExpr (Option α) :=
   let type := toTypeExpr α
   { toExpr     := fun o => match o with
-      | none   => mkApp (mkConst ``Option.none [levelZero]) type
-      | some a => mkApp2 (mkConst ``Option.some [levelZero]) type (toExpr a),
-    toTypeExpr := mkApp (mkConst ``Option [levelZero]) type }
+      | none   => mkApp (mkConst ``Option.none [l.toLevel]) type
```

Comment:
I think the style should be using `toLevel.{u}` rather than naming the `ToLevel` instance:
```suggestion
      | none   => mkApp (mkConst ``Option.none [toLevel.{u}]) type
```